__tests__/player.test.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-env jest */
+const Player = require('../src/player');
+
+describe('Player constructor', () => {
+  it('has a name property',() => {
+    const player = new Player('Clover', jest.fn());
+    expect(player.name).toEqual('Clover');
+  });
+  it('has an empty high scores array', () => {
+    const player = new Player();
+    expect(player._highScores).toEqual([]);
+  });
+});
+
+describe('Is high score method', () => {
+  it('returns true if empty high scores array', () => {
+    const player = new Player('Clover');
+    const _highScores = [];
```

Comment:
In all of these tests, `_highScores` is not linked to the `Player` object at all, so won't have an effect on the tests outcome. Are you meaning to use `player._highScores`?

Currently, all of your tests will be passing because your `player._highScores` is empty.

In this first test, I don't think you need to do anything with `_highScores` anyway - you have already tested that `player._highScores` is empty by default, so you can assume that here.

In the other tests, just change `const _highScores = ... ` to `player._highScores = ...`